locales/it_IT/pcsx2_Main.po

```diff
@@ -2868,39 +2840,39 @@ msgstr ""
 "gzip valido.  Potrebbe essere stato creato da una versione di PCSX2 "
 "precedente e non più compatibile, o potrebbe essere corrotto."
 
-#: pcsx2/gui/SysState.cpp:585
+#: pcsx2/gui/SysState.cpp:583
 msgid "This file is not a valid PCSX2 savestate.  See the logfile for details."
 msgstr ""
 "Questo non è un salvataggio di stato di PCSX2 valido.  Controlla il file di "
-"log per i dettagli."
+"registro per i dettagli."
 
-#: pcsx2/gui/SysState.cpp:604
+#: pcsx2/gui/SysState.cpp:602
 msgid ""
 "This savestate cannot be loaded due to missing critical components.  See the "
 "log file for details."
 msgstr ""
 "Questo salvataggio di stato non può essere caricato a causa di componenti "
-"critici mancanti.  Controlla il file di log per i dettagli."
+"critici mancanti.  Controlla il file di registro per i dettagli."
```

Comment:
C'è un doppio spazio
EDIT: .. che in effetti c'è anche nell'originale, e che è anche presente in molti altri messaggi..

Reply:
bad word